about.html

```diff
@@ -0,0 +1,171 @@
+          to nourish your body while enjoying the foods you love.
+        </p>
+        <p class="third-p">
+          Please contact us per email for any further question about Nutrition
+          <span
+            ><a href="mailto:someone@example.com">nutirstion@gmai.com</a></span
+          >
+        </p>
+      </div>
+      <div class="row">
+        <section class="logo">
+          <h2 class="a">Creative Common<br />Nutrition 2016 Logo</h2>
+          <h2 class="b">Creative Common Nutrition 2016 Logo</h2>
+          <hr />
+          <p>
+            The logo of Nutirstion 2015 was decided <br />
+            through the logo competetion from 8.Jun to 7.July
+          </p>
+          <img src="./images/img4.png" alt="logo" />
+        </section>
+      </div>
+      <div class="row">
+        <section class="healthyRecipes">
+          <div>
+            <h2>See Easy and Cheap Healthy Recipes</h2>
+            <hr />
+            <p class="a">
+              Take a look at the easiest healty recipes which takes <br />
+              no time to prepare it .
+            </p>
+            <p class="b">
+              Take a look at the easiest healty recipes which takes no time to
+              prepare it .
+            </p>
+          </div>
+          <div class="container">
+            <div class="recipes1">
+              <div class="recipeImage">
+                <img
+                  src="./images/Coconut-and-corn-curry-3ef570b.jpg"
+                  alt="Coconuts"
+                  class="sepia"
+                />
+              </div>
+              <div class="centerd">
+                <h3>Coconut & corn curry</h3>
+                <p>
+                  <a href="recipe1.html" target="_blank"> See the Recipe</a>
+                </p>
+              </div>
+            </div>
+            <div class="recipes1">
+              <div class="recipeImage">
+                <img
+                  src="./images/Miso-and-butternut-soup-efe9277.jpg"
+                  alt="Miso"
+                  class="sepia"
+                />
+              </div>
+              <div class="centerd">
+                <h3>Miso & butternut soup</h3>
+                <p>
+                  <a href="recipe2.html" target="_blank">See the Recipe</a>
+                </p>
+              </div>
+            </div>
+          </div>
+        </section>
+      </div>
+      <div class="row">
+        <section class="partner a" id="partner">
+          <h3>Parnter</h3>
+          <hr />
+          <span class="mozilla"> mozilla </span>
+          <span class="google"> Google </span>
+          <span class="naver"> Naver </span>
+          <span class="daumkakao"> daumkakao </span>
+          <span class="airbnb"> airbnb </span>
+        </section>
+      </div>
+      <footer>
+        <div class="left-side">
+          <span> <img src="./images/cc.svg" id="cc" /></span>
+          <p>Creative <br />Commons Korea</p>
+        </div>
+
```

Comment:
- [ ] On the mobile version, as said by the previous reviewer, KIndy make sure your footer section is well-centered as you may notice your design is not.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197973360-3cbcf094-3ee0-4313-b16d-d2e7569961c9.png)

Template design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197973691-ae75965a-e2bb-4ec2-85a9-9eeb93096112.png)

Reply:
I have centered the element in the mobile version.